crates/agglayer-config/tests/snapshots/validate_deserialize__empty_rpcs.snap

```diff
@@ -0,0 +1,60 @@
+---
+source: crates/agglayer-config/tests/validate_deserialize.rs
+expression: config
+---
+prover-entrypoint = "http://127.0.0.1:8080"
+
+[full-node-rpcs]
+
+[l2]
+rpc-timeout = 45
+
+[proof-signers]
+
+[log]
+level = "info"
+outputs = []
+format = "pretty"
+
+[rpc]
+port = 9090
+host = "0.0.0.0"
+request-timeout = 180
+
+[rate-limiting]
+send-tx = "unlimited"
+
+[rate-limiting.network]
+
+[outbound.rpc.settle]
+max-retries = 3
+retry-interval = 7
+confirmations = 1
+settlement-timeout = 1200
+
+[l1]
+chain-id = 1337
+node-url = "http://zkevm-mock-l1-network:8545/"
+ws-node-url = "ws://zkevm-mock-l1-network:8546/"
+rollup-manager-contract = "0xb7f8bc63bbcad18155201308c8f3540b07f84f5e"
+rpc-timeout = 45
+
+[auth.local]
+private-keys = []
+
+[telemetry]
+prometheus-addr = "0.0.0.0:3000"
```

Comment:
Should probably remove this default too, it does not make much sense.

Reply:
it does, it is the metrics entrypoint

Reply:
What does the `0.0.0.0` mean? Does it disable prometheus? If so, I'd probably prefer it to be a bit more explicit about that, using e.g. `null` or `"disabled"` or something.